### PR TITLE
Setting ID DESC order on server, disable reorder on client

### PR DESCRIPTION
**Pull request by mamute**
Fix the bug mentioned at #188

### CODE REVIEW

File: client/npm-shrinkwrap.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
   "version": "1.1.0",
```
Comment (justin808) [inline client/npm-shrinkwrap.json]: Usually package.json changes with the shrinkwrap file.

**Comment by justin808** [comment]
Looking good.

**Comment by adavkay** [comment]
Nice! Looks good to me too.

**Comment by justin808** [comment]
@mamute Please confirm ready to merge.

**Pull request merged**